HIRS_ProvisionerTPM2/build.gradle

```diff
@@ -1,13 +1,22 @@
 task cleanup() {
-    delete 'build'
-}
-
-task prep(type: Exec) {
-    workingDir './'
-    commandLine 'mkdir', 'build'
+    delete 'build/CMakeCache.txt',
```

Comment:
Can you find a way to do this that does not require people to update this list of non`lib` subdirectories of `build`?

Reply:
Think I found a pretty good solution to this.